### DISCUSSION TITLE
personal config in htaccess file

### DISCUSSION THREAD

**Discussion by GitWo262** [FreshRSS/FreshRSS#7145] [Q&A] [answered]
Hi,
I always have problems with the update of FreshRSS and it's just because of my personal modification \p\.htacess file. I need to add there a few lines in front of - which always disappear after an update and this time the update did not work before I did a bigger reconfig of allowed access on my server structure.

Please add the possibility to add a user specific part in fron of the .htaccess file.

Thanks Wolle

**Comment by Alkarex** [comment, marked as answer]
If possible, place your personal `.htaccess` in `./FreshRSS/p/i/.htaccess` or in `./FreshRSS/.htaccess`
You are not supposed to edit `./FreshRSS/p/.htaccess`

**Reply by GitWo262** [reply]
Thanks for your quick replay. 
I did now a complete reconfiguration of my server setup and now there is no more any need for a personal  htaccess inside the FreshRSS structure. Thanks for your impulse.

**Reply by Alkarex** [reply]
https://github.com/FreshRSS/FreshRSS/pull/7153
